wiki/Client/Program_files/de.md

```diff
@@ -0,0 +1,110 @@
+
+### Songs
+
+Der Ordner Songs speichert die osu!-Beatmaps des Spielers. Er enthält in der Regel Dateien mit den Formaten `.osu` (Schwierigkeitslevel), `.mp3`/`.ogg` (Musikdateien), `.jpg`/`.png`/`.gif` (Hintergrundbilder), `.osb` (Storyboard-Dateien) und `.mp4`/`.flv` (Videodateien). Auch können Dateien mit dem Format `.wav`/`.ogg` (Hitsound-Dateien) und Ordner (Storyboard-Sprites und/oder Skin-Ordner) vorhanden sein.
+
+Die Namensstruktur einer Datei ist `{Beatmap-ID} {Künstler} - {Songtitel}`.
+**Beispiel:** [57950 SOUND HOLIC - Drive My Life](https://osu.ppy.sh/beatmapsets/57950)
+
+Bitte beachte, dass einige sehr alte Beatmaps (zum Beispiel [Kenji Ninuma - DISCO PRINCE](https://osu.ppy.sh/beatmapsets/1) oder [Dudelstudios - Angry Video Game Nerd Theme [MATURE CONTENT]](https://osu.ppy.sh/beatmapsets/66)), sowie nicht eingereichte Beatmaps, nicht diesem Format folgen.
+
+## Versteckte Ordner
+
+Diese Ordner werden ausgeblendet, weil jede Änderung dazu führen könnte, dass osu! nicht mehr richtig oder überhaupt nicht mehr startet.
+
+### Data
+
+Die Datenbestände von osu!. Enthält den Cache von osu!, wie beispielsweise zu Beatmap-Hintergründen und Avataren. Der Cache sollte nicht gelöscht werden, da er möglicherweise von osu! verwendet wird.
+
+## Dateien
+
+*Vorsicht: Sei vorsichtig bei diesen Dateien! Du könntest osu! kaputt machen, wenn du nicht behutsam damit umgehst.*
+
+### Datenbankdateien (.db)
+
+Die Datenbankdateien sind Datenbanken, die osu! benötigt, um richtig zu funktionieren. Die Dateien enthalten wesentliche Informationen, die osu! braucht, wie zum Beispiel hinterlegte Scores und die zwischengespeicherte Liste der Beatmaps, die auf dem Gerät des Spielers vorhanden sind.
+
+- `collections.db`: Verwaltet die Sammlungen des Spielers im Spiel.
+- `osu!.db`: Speichert die Beatmap-Datenbank von osu!.
+- `presence.db`: Führt einen Cache der osu!-Spieler, die in der Chatkonsole eingeloggt sind.
+- `scores.db`: Enthält die lokalen Bestenlisten.
+
+### .cfg (Konfigurationsdateien)
+
+Konfigurationsdateien stellen die Standardeinstellungen bereit, damit osu! funktioniert. Die Dateien können mit einem Texteditor geöffnet werden.
+
+- `osu!.cfg`: Speichert Sicherheitsinformationen über die osu!-Anwendungsdateien und die aktuelle Updatequelle. Dies sollte niemals manuell bearbeitet werden.
+- `osu!.<Benutzername im Betriebssystem>.cfg`: Speichert Daten zu den [Optionen](/wiki/Client/Options) und anderen Spieleinstellungen (siehe [Nutzerkonfigurationsdatei](/wiki/Client/Program_files/User_configuration_file)).
+
+### .exe (Anwendung)
+
+Das Kernstück. Klicke darauf, um es zu starten (gilt nur für Windows). Die .exe-Dateien können ohne Bedenken geöffnet werden, vorausgesetzt, es wurde das osu!-Installationsprogramm von der offiziellen Webseite verwendet.
+
+osu!.exe (Starte osu!)
+
+### .dll (Anwendungserweiterung)
+
+Diese .dll-Dateien sind die Komponenten und Dependencies von osu!.
```

Comment:
I wanted to find an alternative for `Dependency` at first, and I think that some might use `Abhängigkeit`. Though, Dependency is still pretty common, so I'm going with this.